### DISCUSSION TITLE
Sequencing depth corrected counts

### DISCUSSION THREAD

**Discussion by faraz89** [campbio/celda#328] [Q&A]
Hi,

Would it be acceptable to use sequencing depth corrected counts (NOT log transformed) data as input for decontX? Apparently I tested decontx on raw counts and sequencing depth corrected counts and is apparently performing way better with corrected counts than raw. 

Thanks,
Faraz

**Comment by Irisapo** [comment]
Hi @faraz89 , 

Interesting. What sequencing depth correction did you use? Is the total UMIs after correction for a cell 1 or other fixed number?

It is totally reasonable to use sequencing depth corrected counts data as input for decentX. The better performance could be that the contamination distribution corresponding for each cell group (potentially a cell type) is not affected by the sequencing depth as a consequence. But just be aware, applying sequencing depth correction can also removes important biological information such as the overall activity of the cell that can be measured by total number of UMIs. And this kind of biological information plays a part in performance using DecontX. 

I also would suggest the number being some reasonable number rather than 1 if sequencing depth correction results into a fixed number of total UMIs for each cell. 

Let me know if this makes sense and if anything else I can answer.
Shiyi

**Comment by faraz89** [comment]
Many thanks Shiyi (@Irisapo) for getting back to me.

The correction is based on Seurat SCTransform function which uses Pearson residuals from “regularized negative binomial regression,” where cellular sequencing depth is utilized as a covariate in a generalized linear model (GLM). The parameters for the model are estimated by pooling information across genes that are expressing at similar levels. This should remove the technical characteristics but preserve the biological heterogeneity, and avoid overfitting the model to the data.

Thanks,
Faraz

**Comment by Irisapo** [comment]
Hi Faraz @faraz89 , 

It makes sense to correct sequencing depth and use the results as input for decentX from what you described here on SCTransform. 
If I find any caveat later on when I dig more into their correction method I will let you know. 

Best, 
Shiyi

**Comment by joshua-d-campbell** [comment]
Hi @faraz89,
We haven't thoroughly tested this, and so we can't really say whether it is a good idea or not. Our original approach was benchmarked on several datasets where the ground truth was known or the biology was very clear. When you say it is "performing better on the corrected counts", how are you measuring performance in this scenario? In the meantime, we will try to take a look at the idea using our benchmarking datasets.

**Comment by faraz89** [comment]
Hi Joshua (@joshua-d-campbell),

The performance was assessed based on much the contaminants were removed while maintaining the real counts. In the sequencing depth corrected counts, more contaminants were removed while real counts were intact. In addition, the seurat analysis showed a better separation of distinct cell types based on clustering analysis.
